external/kafka/src/main/scala/org/apache/spark/streaming/kafka/KafkaTestUtils.scala

```diff
@@ -227,10 +229,33 @@ private class KafkaTestUtils extends Logging {
     tryAgain(1)
   }
 
+  def waitUntilLeaderOffset(
```

Comment:
What does this function do? Either make the name more meaningful (like `waitUntilMetadataIsPropagated`) or add scala docs.